### ISSUE TITLE
Wayland please!

### ISSUE THREAD

**Issue by yemouu** [BanchouBoo/boox#1]
Thank you for reading :)

**Comment by BanchouBoo** [comment]
I don't know anything about programming Wayland stuff, perhaps one day but probably not for awhile if I ever decide to port it. If you have any resources you could point me to where I could learn about how I might begin to implement something like this I'd be happy to take a look.

**Comment by BanchouBoo** [comment]
Soon:tm: :eyes:

**Comment by yemouu** [comment]
pog

**Comment by BanchouBoo** [comment]
Still needs a lot of work, but here's it running in Sway

https://user-images.githubusercontent.com/32691832/156974311-8091c758-4fe0-48ab-ac40-94a8b9087ec2.mp4

**Comment by yemouu** [comment]
oh nice. are you rewriting it in zig?

**Comment by BanchouBoo** [comment]
yeah, it'll be in a new repo once it's done. getting very close, at this point I'm just cleaning up code and fixing some minor issues I've come across. expect it in like a week at the longest assuming I don't discover some drastically horrible problem with my code.

**Comment by BanchouBoo** [comment]
Evidently I never got around to doing that, I got burnt out on it but I finally cleaned some stuff up and created a repo for it. Wayland backend still needs work, I won't make any promises this time on when that will get done, however. 

https://github.com/BanchouBoo/bento
